api/utils/aws/endpoint.go

```diff
@@ -59,6 +59,12 @@ func IsMemoryDBEndpoint(uri string) bool {
 	return isAWSServiceEndpoint(uri, MemoryDBSServiceName)
 }
 
+// IsKeyspacesEndpoint returns true if input URI is an AWS Keyspaces endpoint.
+// https://docs.aws.amazon.com/keyspaces/latest/devguide/programmatic.endpoints.html
+func IsKeyspacesEndpoint(uri string) bool {
+	return strings.HasPrefix(uri, "cassandra.") && IsAWSEndpoint(uri)
```

Comment:
There's also `cassandra-fips` (fedramp/govcloud only?)
Should we relax the condition to `"cassandra"` or add an an `OR` to `"cassandra-"` ?

Reply:
good catch! will fix